src/protagonist/API/Features/Storage/Converters/CustomerStorageConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace API.Features.Storage.Converters;
+
+/// <summary>
+/// Conversion between API and EF forms of CustomerStorage resource
+/// </summary>
+public static class CustomerStorageConverter
+{
+    /// <summary>
+    /// Convert CustomerStorage entity to API resource
+    /// </summary>
+    public static DLCS.HydraModel.CustomerStorage ToHydra(this DLCS.Model.Storage.CustomerStorage customerStorage, string baseUrl)
+    {
+        var hydraCustomerStorage = new DLCS.HydraModel.CustomerStorage(baseUrl, customerStorage.Customer, customerStorage.Space)
+        {
+            StoragePolicy = $"{baseUrl}/storagePolicies/{customerStorage.StoragePolicy}",
```

Comment:
Does this happen elsewhere? IIRC the `HydraModel` should set any links to other API endpoints (I might be wrong so double check).

Reply:
See `CustomerOriginStrategyConverter`: https://github.com/dlcs/protagonist/blob/af0a69377ce47c19b7225fcc71aa837daff67178/src/protagonist/API/Features/OriginStrategies/Converters/CustomerOriginStrategyConverter.cs#L25

I think we talked about this while I was working on https://github.com/dlcs/protagonist/issues/541 - AFAIK, it wouldn't automatically set the link there either, so I was suggested to set the path manually in the constructor.